cmd/kubeadm/app/cmd/init.go

```diff
@@ -31,15 +31,16 @@ import (
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmapiext "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1alpha1"
 	"k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/validation"
+	cmdphases "k8s.io/kubernetes/cmd/kubeadm/app/cmd/phases"
```

Comment:
I would argue there is no need to have the `phase` package and we should just have a `cmdphase` package (or maybe abbreviate phase to `ph`) and all the other ones we are using (cert phase, etc.).  

Having the `phase` package just be a name holder doesn't seem prudent.